core/src/Cuda/Kokkos_CudaSpace.cpp

```diff
@@ -45,17 +45,17 @@
 #include <Kokkos_Macros.hpp>
 #ifdef KOKKOS_ENABLE_CUDA
 
+#include <Kokkos_Core.hpp>
+#include <Kokkos_Cuda.hpp>
+#include <Kokkos_CudaSpace.hpp>
```

Comment:
I assume this came from format?

Reply:
No, this is because system headers are supposed to come after library headers, so as to avoid polluting the latter. Because I originally had my template implementation file up here as well, I moved these when I put it in here. If you'd like me to revert I'm happy to do so.

Reply:
No I think this is right, we just are always trying to decide when to do things like formatting or other clean up.